modules/dxp/apps/portal-workflow/portal-workflow-metrics-rest-test/src/testIntegration/java/com/liferay/portal/workflow/metrics/rest/resource/v1_0/test/InstanceResourceTest.java

```diff
@@ -93,8 +93,20 @@ public void testGetProcessInstance() throws Exception {
 		instance.setSlaResults(
 			new SLAResult[] {
 				_toSLAResult(true, SLAResult.Status.NEW),
+				_toSLAResult(true, SLAResult.Status.NEW),
```

Comment:
Hey @seltong, I think that create only 9 or 10 slaResults is enough to your test case since we need just to assert that we are retrieving more than 10 documents from elastic in the Instance query that you modified. 

Thx